installer/Util/AppSetupUtility.php

```diff
@@ -456,6 +456,19 @@ public function writeConfFile(): void
         clearstatcache(true);
     }
 
+    public function writeKeyFile(): void
+    {
+        $cryptKey = '';
+        for ($i = 0; $i < 4; $i++) {
+            $cryptKey .= md5(rand(10000000, 99999999));
```

Comment:
read the Caution https://www.php.net/manual/en/function.rand.php
use https://www.php.net/manual/en/function.random-bytes.php,
`bin2hex(random_bytes(...))`
may have to do something to backward compatible with 4.x.

Reply:
changed to `random_int`